employee-onboarding-A2A-reference-implementation/src/auth/asgardeo.py

```
@@ -0,0 +1,649 @@
+            ).decode()
+            
+            response = await client.post(
+                self.settings.asgardeo_token_url,
+                data=data,
+                headers={
+                    "Content-Type": "application/x-www-form-urlencoded",
+                    "Authorization": f"Basic {basic_auth}"
+                }
+            )
+            
+            vlog(f"\n[RESPONSE]")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)